test-commands.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+set -e
+
+mkdir testNeuCLI
+cd testNeuCLI
+
+# NEU
+
+commandRun(){
+    echo -e "\e[1;32mCommand Run: $1\e[0m"
+}
+
+echo -e "\e[1;32mTesting Neutralino CLI commands\n"
```

Comment:
Sounds we can make a function called `echo_green` and add this colorized logic? Then, we can re-use the same function in the `display_cmd` too I guess